docs/Documentation/Development/Programming_Models/mpi.md

```diff
@@ -1,6 +1,57 @@
-* Cray MPICH
-* Open MPI
-* Intel MPI
-* MPI in NVHPC 
+# OpenMpi
```

Comment:
I think we should title the page more generally, like "MPI", and then make the different implementation headers as subsections (e.g. "##"). Subsections will render in the table of contents which I think is helpful for reader navigation.